### DISCUSSION TITLE
Having trouble claiming Student Dev Pack benefits

### DISCUSSION THREAD

**Discussion by ia74** [community/community#69255] [GitHub Education]
### Hi everyone,

I've got the "Enjoy the benefits of being a student" when I press benefits, and a purple Approved marker on the application page. I am unable to access the Student Developer Pack and any of it's benefits, I get the page with "We're still processing your application. Once this is complete you will have access to the student developer pack offers.". Pressing the button takes me back to the application page. If I try resubmitting my information, it tells me that I don't need to as I was already verified on September 30, 2023.

**Comment by saher2106** [comment]
> ### Hi everyone,
> I've got the "Enjoy the benefits of being a student" when I press benefits, and a purple Approved marker on the application page. I am unable to access the Student Developer Pack and any of it's benefits, I get the page with "We're still processing your application. Once this is complete you will have access to the student developer pack offers.". Pressing the button takes me back to the application page. If I try resubmitting my information, it tells me that I don't need to as I was already verified on September 30, 2023.
> 
> ### A suggestion...(optional)
> _No response_

you should check it out in the settings, go to billings and plans-->plans and usage and you can check if it is activated or not

**Reply by Life45** [reply]
> you should check it out in the settings, go to billings and plans-->plans and usage and you can check if it is activated or not

I'm having the exact same problem and it doesn't show up in plans and usage as well.

**Reply by saher2106** [reply]
you can contact github support from the settings, it might help!

**Reply by ia74** [reply]
It’s not activated there

**Comment by Akbaroke** [comment]
It sounds like your GitHub Student Developer Pack application has been approved, but you are still having trouble accessing the benefits. This is likely due to a temporary delay in processing your application.

Here are a few things you can try:

Wait a bit longer. It may take some time for your application to be fully processed and for you to be granted access to the benefits.
Try logging out of GitHub and then logging back in. This can sometimes help to refresh your permissions.
Try clearing your browser's cache and cookies. This can sometimes fix problems with websites.
Contact GitHub support for help. If you are still having trouble after trying the above, you can contact GitHub support for help.
Here is an optional suggestion:

Try using a different browser or device. It is possible that there is a problem with your current browser or device that is preventing you from accessing the benefits.
I hope this helps!

**Comment by sinister-virus** [comment]
I have not received any benefits in the last 10 months.
The approved is Purple, but why it hasn’t turned green yet?
![Screenshot 2024-04-26 154421](https://github.com/community/community/assets/51523645/b095a030-8cf1-4b4a-b59d-ecb96725a3fd)
![Screenshot 2024-04-26 154434](https://github.com/community/community/assets/51523645/592a1319-761d-488e-9750-163384e8dbb3)

**Reply by GinoongFlores** [reply]
As far as I know, the green indicator is for pending application. I have mine approved with that indicator, so you're good to go now to have the benefits.
